### ISSUE TITLE
[feature request] properties _NET_WM_DESKTOP _NET_FRAME_EXTENTS to be setted

### ISSUE THREAD

**Issue by paoloschi** [JLErvin/berry#25]
In _berry_, if I run this command...

```
$ xprop -id 0xe00006 _NET_WM_DESKTOP _NET_FRAME_EXTENTS
_NET_WM_DESKTOP:  not found.
_NET_FRAME_EXTENTS:  not found.
```

I can't migrate some of my automation scripts without them; in particular I miss the first one without which I don't know how I could otherwise have a complete overview of the position of all windows on the different workspaces.
The second one can be used to determine the current status "decorated == yes|no"
I don't know how else to get such values and I would still like to avoid third party tools (_xdotool_, _wmctrl_....) and be able to continue to use only _xprop_ and _xwininfo_...

**Comment by JLErvin** [comment]
Just added these to the development branch [blackberry](https://github.com/JLErvin/berry/tree/blackberry). Although I'm a bit confused by your description of this protocols. According to the [ewmh](https://standards.freedesktop.org/wm-spec/wm-spec-latest.html#idm140200472552416) documentation, `_NET_WM_DESKTOP` should report the current desktop of the given client and `_NET_FRAME_EXTENTS` should report the region occupied by the current client. Is this what you meant, or am I misunderstanding the requirements of this protocols?

**Comment by paoloschi** [comment]
My ability to express myself clearly in English is limited and I apologize for that (I often have to use an automatic translator :-[).

In order to be sure that I was making a correct request, I consulted the same document just yesterday. Regarding the two specific properties he recommends, among other things, that "_The Window Manager should honor_ _NET_WM_DESKTOP..." and also "_The Window Manager MUST set_ _NET_FRAME_EXTENTS..." so I thought it was appropriate to make the request.

About _NET_WM_DESKTOP which "_determine the desktop the window is in_" I confirm: it is exactly the data I need.

As the document says, _NET_FRAME_EXTENTS "_are widths of the respective borders added by the Window Manager_"; I'm interested in getting the status of the window about whether or not it's currently decorated otherwise my script can't know if it's the case or not to run a `berryc toggle_decorations` (...). Since there is no specific EWMH property about the current status of the decoration, _NET_FRAME_EXTENTS can become a way to deduce it.
Alternatively, for the complete supply of the status of a window, a proprietary solution as in _Windowchef_ (a request that we've already discussed [here](https://github.com/JLErvin/berry/issues/2#issuecomment-504780683)) would be optimal.

The same command executed under Openbox:
```
$ xprop -id 0x1600007 _NET_WM_DESKTOP _NET_FRAME_EXTENTS
_NET_WM_DESKTOP(CARDINAL) = 3
_NET_FRAME_EXTENTS(CARDINAL) = 6, 6, 24, 6
```

Thank you very much for your prompt response and attention to these requests of mine!
I promise not to make any more immediate requests: I will enthusiastically follow the project according to your development plans :-)
